### DISCUSSION TITLE
Is it possible to work with url paths instead different ports?

### DISCUSSION THREAD

**Discussion by ghost** [hoppscotch/hoppscotch#3644] [Q&A] [answered]
We are currently evaluate/test hoppscotch as postman alternative and it should run on our k8s plattform. 
We did't want to change our ingess controller to listen on other ports as 80 or 443.
In the first step we create a ingress controller with 3 hostnames
ex. hoppscotch, hoppscotch-admin & hoppscotch-backend
On the first look it works but it is not "beautiful"

From our point of view it looks better when we can use url paths like hoppscotch/admin hoppscotch/backend.
Can be this a problem in later use cases when our dev teams are working with the tool?
Or trigger this change other unexpected behavior?

**Comment by liyasthomas** [comment]
We're working on implementing the ability to change ports to use different subpaths hence, please stay patient for our December release. I'll update here once it's available in Docker Hub.

**Reply by ghost** [reply]
Perfekt

**Comment by liyasthomas** [comment, marked as answer]
This feature is now available in the latest release candidate: v2023.12.0.

Read our updated self-hosting documentation: https://docs.hoppscotch.io/documentation/self-host/getting-started
